src/shared/components/VideoPreviewBase/VideoPreviewBase.styles.tsx

```diff
@@ -208,22 +219,49 @@ export const CoverVideoPublishingStateOverlay = styled.div`
   padding: ${sizes(1)} ${sizes(2)};
   display: flex;
   align-items: center;
-  background-color: ${colors.gray['900']};
-  opacity: 0.7;
+  background-color: ${colors.black};
+  color: ${colors.white};
+  z-index: ${zIndex.overlay};
 `
 
 export const PublishingStateText = styled(Text)`
   margin-left: ${sizes(1.5)};
 `
 
+export const KebabMenuIconContainer = styled.div<{ isActive?: boolean }>`
+  display: ${({ isActive }) => (isActive ? 'flex' : 'none')};
+  align-items: center;
+  justify-content: center;
+  width: 36px;
+  height: 36px;
```

Comment:
let's maybe use `sizes` here
```suggestion
  width: ${sizes(9}};
  height: ${sizes(9}};
```